src/translators.js

```diff
@@ -1,78 +1,64 @@
 /*
-    ***** BEGIN LICENSE BLOCK *****
```

Comment:
Revert unrelated whitespace changes

(You'll want to configure your editor not to reindent the whole file.)